packages/graphql-language-service-server/src/index.ts

```diff
@@ -16,4 +16,3 @@ export { Logger } from './Logger';
 export * from './GraphQLCache';
 export * from './parseDocument';
 export * from './findGraphQLTags';
-export * from './stringToHash';
```

Comment:
this export does nothing since `stringToHash` export default export only
![image](https://user-images.githubusercontent.com/7361780/204149231-455283ef-75ae-470c-a595-bd43745f35dd.png)

Reply:
sounds good to me